FeedStoreChallenge/CoreDataModels/ManagedFeedImage.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Husam Dayya on 17/03/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedFeedImage)
+public class ManagedFeedImage: NSManagedObject {
+	@NSManaged public var id: UUID
+	@NSManaged public var imageDescription: String?
+	@NSManaged public var location: String?
+	@NSManaged public var url: URL
+	@NSManaged public var cache: ManagedCache
+}
```

Comment:
This class and all its properties and methods shouldn't be public. They're internal implementation details.

```suggestion
class ManagedFeedImage: NSManagedObject {
	@NSManaged var id: UUID
	@NSManaged var imageDescription: String?
	@NSManaged var location: String?
	@NSManaged var url: URL
	@NSManaged var cache: ManagedCache
}
```